disnake/embeds.py

```diff
@@ -174,7 +180,9 @@ class Embed:
         "_files",
     )
 
+    default_colour: Union[int, Colour] = Colour(0)
     Empty: Final = EmptyEmbed
+    default_color: Union[int, Colour] = alias_of("default_colour")
```

Comment:
type-checkers won't like this, maybe have a separate function that proxies `alias_of` but with an `Any` return type annotation? (similar to `Param` and `ParamInfo`)

Reply:
Removed this in change for class methods.